src/main/java/com/google/step/youtube/DescriptionParser.java

```diff
@@ -52,14 +54,34 @@ public Pattern getPattern() {
      * @param description of the YouTube video to be parsed.
```

Comment:
nit .  if you decide to use the whole `@param`/`@return `thing, include all the args.  basically specify all or none.  (you're missing `@param` pattern)

Reply:
Is this for the findMatches() javadoc? `@param pattern` is on line 51 hidden by git.